modules/tinymce/CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The dialog size was not updated when the `size` argument was changed when redialling a dialog. #TINY-10209
 - Toggling a list that contains an LI element having another list as its first child would remove the remaining content within that LI element. #TINY-10213
 - Custom block element wasn't considered block element in some cases. #TINY-10139
+- Search and replace plugin would incorrectly find matching text inside non-editable root elements. #TINY-10162
+- Search and replace plugin would incorrectly find matching text inside SVG elements. #TINY-10162
```

Comment:
Once we introduce SVG support in this release then maybe we should not add an entry for the bug here

Reply:
It kind of is supported if you configure the schema in previous versions. We are just making it easier now to opt out of all filtering inside svg elements without having a massive schema. If you manually configure svg like:
```
	extended_valid_elements: 'svg[*],circle[*]',
	valid_children: 'svg[circle|#text],circle[#text]',
	custom_elements: 'svg'
```

It would work in the current released version. So search replace would fail in that case.